app/controllers/hiring_staff/schools_controller.rb

```diff
@@ -32,4 +32,21 @@ def update
   def session_has_multiple_schools?
     session.key?(:multiple_schools) && session[:multiple_schools] == true
   end
+
+  def set_vacancies
```

Comment:
I remember you mentioning that you thought the use of a switch statement here wasn't the most elegant. 

Now we've got this in a good place perhaps a way to refactor this would be to introduce a small and testable presenter object that would move this logic away from the controller.

```
@school_vacancy_presenter = SchoolVacancyPresent.new(type: params[:type])
@school_vacancy_presenter.draft
@school_vacancy_presenter.pending
@school_vacancy_presenter.expired
@school_vacancy_presenter.live
@school_vacancy_presenter.current_type
```

`SchoolVacancyPresenter` is a bit long, but we already have a `VacancyPresenter` for the job seekers!